R/inboveg_classification.R

```diff
@@ -0,0 +1,51 @@
+#' Provide vegetation classifications
+#'
+#' Get the available vegatation classifications for the vegetation surveys
+#'
```

Comment:
based on European classification (e.g. N2000) and local classification (e.g. BWK), together with the percentage cover and certainty.